### DISCUSSION TITLE
Problem with RG350-folders

### DISCUSSION THREAD

**Discussion by Mastelloni** [eduardofilo/RG350_adam_image#96] [SD topics]
I did the step by step to put the first image, I have a RG350, and even putting the two cards the external card doesn't have the folders...I tried to put the folders manually on the external card along with the ROMS, and even so it didn't work. ..

The external card was formatted as FAT32, if I format the internal card, what format should it be?

I believe this is the problem...

**Comment by eduardofilo** [comment]
The two keys are:
* FAT32 format
* Empty label

So if you say that are already using FAT32, the problem could be the label. Double check that:

https://github.com/eduardofilo/RG350_adam_image#external-microsd-format-and-label

**Comment by Mastelloni** [comment]
> As duas chaves são:
> 
> * Formato FAT32
> * Etiqueta vazia
> 
> Portanto, se você disser que já está usando FAT32, o problema pode ser o rótulo. Verifique se:
> 
> https://github.com/eduardofilo/RG350_adam_image#external-microsd-format-and-label


I already formatted it in FAT32, but I had deleted the micro sd partition, I don't know if it interferes... it is also unnamed.

Should I redo the image process for the internal card or would the error be on the external card?

**Comment by eduardofilo** [comment]
Deleted the micro sd partition? I'm not sure I understand that.
After flashing the internal card with the image, you do not have to touch anything there so that the external one can be read.
If there are problems reading the external card, the problems are there.
In addition to what was said before (the format and the label), you may have problems with that particular card. Try another or clean the contacts with alcohol.

**Comment by Mastelloni** [comment]
> Excluiu a partição micro SD? Não tenho certeza se entendi isso.
> Depois de piscar o cartão interno com a imagem, você não precisa tocar em nada para que o cartão externo possa ser lido.
> Se houver problemas de leitura da placa externa, os problemas estão aí.
> Além do que foi dito antes (o formato e a etiqueta), você pode ter problemas com esse cartão específico. Experimente outro ou limpe os contatos com álcool.

This, I had deleted the partition and then created a new partition and then formatted it in FAT32, I will try another card and see if it will work.
Thanks for answering!

**Comment by Mastelloni** [comment]
> Deleted the micro sd partition? I'm not sure I understand that.
> After flashing the internal card with the image, you do not have to touch anything there so that the external one can be read.
> If there are problems reading the external card, the problems are there.
> In addition to what was said before (the format and the label), you may have problems with that particular card. Try another or clean the contacts with alcohol.

![image](https://user-images.githubusercontent.com/90715618/133347240-9b78b677-ea37-4936-8a7e-5fb1a4ecfdb2.png)


I've done everything, and apparently the external card is normal...

**Comment by Mastelloni** [comment]
> Deleted the micro sd partition? I'm not sure I understand that.
> After flashing the internal card with the image, you do not have to touch anything there so that the external one can be read.
> If there are problems reading the external card, the problems are there.
> In addition to what was said before (the format and the label), you may have problems with that particular card. Try another or clean the contacts with alcohol.

What should be the formatting of the internal card that has the image?
I've already tested 3 microsd to stay on the outside and the folders that should have never appear...

**Comment by eduardofilo** [comment]
Your EXT card seems well formatted and without the label.
The INT card, if the console boots, it is OK also.
Open Commander and navigate to path /media/sdcard. Put a snapshot here.

**Comment by Mastelloni** [comment]
> Seu cartão EXT parece bem formatado e sem a etiqueta.
> O cartão INT, se o console inicializar, também está OK.
> Abra o Commander e navegue até o caminho / media / sdcard. Coloque um instantâneo aqui.

The photo follows, the external card has nothing because the folders that should have been created were not created...

![image](https://user-images.githubusercontent.com/90715618/133475054-b968babc-0448-450f-949c-ecc99b8ea698.png)

**Comment by Mastelloni** [comment]
> Your EXT card seems well formatted and without the label.
> The INT card, if the console boots, it is OK also.
> Open Commander and navigate to path /media/sdcard. Put a snapshot here.

Could you help me, please?

**Comment by eduardofilo** [comment]
I don't have the console handy now, but I'll give you more instructions in a few hours.

**Comment by Mastelloni** [comment]
> I don't have the console handy now, but I'll give you more instructions in a few hours.

OK thank you!

**Comment by eduardofilo** [comment]
Do you know how to open an SSH session against the console connected by USB cable to the computer?

**Comment by eduardofilo** [comment]
If you could manage to do it it would help, but in the meantime do the following:

1. In SimpleMenu press Start.
2. Change `Default launcher` to `no`.
3. In the same menu, select `Session quit`. Now you are in GMenu2X.
4. Open `Commander` application.
5. Navigate to `/proc/` directory. Tip: pressing `B` you go one directory up.
6. Open file `mounts` in that directory. i.e. `/proc/mounts`.
7. Send me a picture of what you see.

If you want to go back to SimpleMenu as default launcher, open it, and select `yes` in `Default launcher`.

**Comment by eduardofilo** [comment]
Do one more thing. Mount the external card on your computer and copy any file to it. Then repeat the photo with Commander on the `/media/sdcard` directory.

**Comment by Mastelloni** [comment]
> Você sabe como abrir uma sessão SSH no console conectado por cabo USB ao computador?

Would it be in any of these accesses?

![image](https://user-images.githubusercontent.com/90715618/133664017-094b242e-2ad5-45de-ac69-b3a10df955c3.png)

**Comment by Mastelloni** [comment]
> Se você pudesse fazer isso, ajudaria, mas enquanto isso, faça o seguinte:
> 
> 1. Em SimpleMenu, pressione Iniciar.
> 2. Mude `Default launcher`para `no`.
> 3. No mesmo menu, selecione `Session quit`. Agora você está no GMenu2X.
> 4. Abra o `Commander`aplicativo.
> 5. Navegue até o `/proc/`diretório. Dica: pressionando `B`você sobe um diretório.
> 6. Abra o arquivo `mounts`nesse diretório. ie `/proc/mounts`.
> 7. Envie-me uma foto do que você vê.
> 
> Se você quiser voltar ao SimpleMenu como iniciador padrão, abra-o e selecione `yes`em `Default launcher`.

Here are the photos.

![image](https://user-images.githubusercontent.com/90715618/133664139-7bd869fc-732f-4f74-8f19-0be5e58cd595.png)
![image](https://user-images.githubusercontent.com/90715618/133664188-c1bab280-09b9-4da6-a90b-5653fcedd3ab.png)

**Comment by Mastelloni** [comment]
> Faça mais uma coisa. Monte o cartão externo em seu computador e copie qualquer arquivo nele. Em seguida, repita a foto com Commander no `/media/sdcard`diretório.



> Faça mais uma coisa. Monte o cartão externo em seu computador e cópia qualquer arquivo nele. Em seguida, repita a foto com Commander no `/media/sdcard`diretório.

I put a GBC rom in two formats, but it doesn't show up...

![image](https://user-images.githubusercontent.com/90715618/133666986-15f76b78-f7b3-4acf-9244-a6612f4fc922.png)

![image](https://user-images.githubusercontent.com/90715618/133667019-001eb1df-27fb-4e11-8380-37ddd0d1466f.png)

**Comment by eduardofilo** [comment]
That is the problem:

![133664188-c1bab280-09b9-4da6-a90b-5653fcedd3ab](https://user-images.githubusercontent.com/2923728/133678910-f4b38e1b-034e-4ff8-a2e0-0b76ae1d846f.png)

If you see, the partition is not mounted on `/media/sdcard` but on `/media/mmcblk1p5`. That is why your system doesn't create the folders or can't read the files you put in there.

You have somehow created a strange partition structure on that card. It should look like this:

![screenshot003](https://user-images.githubusercontent.com/2923728/133679170-d3a43696-0674-4dec-80bc-219265a93c5d.png)

I don't use Windows so I don't have a good utility controlled, but it sounds to me that this is used regularly:

https://www.sdcard.org/downloads/formatter/

**Comment by Mastelloni** [comment]
So should I download this app and format the external card in FAT32?

Do I need to redo the image of the internal card? Can I put the internal image as 1.2?

**Comment by eduardofilo** [comment]
All the problems are related to external card. You don't need to do anything with the internal one, although if you still have it with 1.1 or 1.0.3, it is a good time to flash 1.2.

Yes, download that app, to see if it helps you.

**Comment by Mastelloni** [comment]
> All the problems are related to external card. You don't need to do anything with the internal one, although if you still have it with 1.1 or 1.0.3, it is a good time to flash 1.2.
> 
> Yes, download that app, to see if it helps you.

When I used the application you said, it formatted the card in exFAT, so I used a new program called EaseUS Partition Master 15.8, and then I formatted it again in FAT32, and put it in RG350 and now it created the folders!!!

Thank you very much for your help, I will update the internal card with the new improved image, thanks a lot!

**Comment by Mastelloni** [comment]
> Todos os problemas estão relacionados à placa externa. Você não precisa fazer nada com o interno, embora se ainda o tiver com 1.1 ou 1.0.3, é uma boa hora para atualizar o 1.2.
> 
> Sim, baixe esse aplicativo, para ver se te ajuda.

One doubt, yesterday I tried to open a nintendo game watch game and it gave a problem in the resolution, then I turned it off using the power, today I tried to turn it on it stays on this screen, it's blinking.
I took the battery out and put it back in, and I opened the kernel on the PC and put it back in as rg350 but it doesn't turn on, I'm using the 1.2 image.

![image](https://user-images.githubusercontent.com/90715618/133784000-3cd892af-27de-4663-993f-2e2acb794af5.png)


**Can you help me?**

**Comment by eduardofilo** [comment]
OK. So the problem was the EXT SD format. I close this issue that will serve as example of the situation for more people.

**Comment by Mastelloni** [comment]
> OK. Portanto, o problema era o formato EXT SD. Fecho este assunto que vai servir de exemplo da situação para mais pessoas.

Yeah, could you see my other problem that I posted above?

**Comment by eduardofilo** [comment]
This problem it is not related with an issue with the image. Better ask in forums (I take part in [this Discord server](https://discord.com/channels/529983248114122762/871075357422415952)).

G&W tends to give that resolution problems. See this tips when happens: https://github.com/eduardofilo/RG350_adam_image#unsupported-video-mode

If you forced the boot of the console with reset, surelly the internal card will have been corrupted, so the solution it is to reflash that card. Remember, the internal card this time.

**Comment by Mastelloni** [comment]
> Este problema não está relacionado com um problema com a imagem. Melhor perguntar em fóruns (eu participo [deste servidor Discord](https://discord.com/channels/529983248114122762/871075357422415952) ).
> 
> G&W tende a apresentar problemas de resolução. Veja estas dicas quando acontecer: https://github.com/eduardofilo/RG350_adam_image#unsupported-video-mode
> 
> Se você forçou a inicialização do console com reset, provavelmente a placa interna terá sido corrompida, então a solução é atualizar a placa novamente. Lembre-se, o cartão interno desta vez.

Okay, I'll check it out, thanks.
